kotlin-jvm/src/main/kotlin/playground/SQLDelight.kt

```diff
@@ -0,0 +1,88 @@
+@file:Suppress("PackageDirectoryMismatch")
+
+package playground.sqldelight
+
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
+import playground.shouldBe
+import util.AppDatabase
+import util.Orders
+import util.User
+import java.util.*
+
+/**
+ *  SQLDelight - Generates typesafe Kotlin APIs from SQL
+ *  - GitHub: https://github.com/cashapp/sqldelight
+ *  - Official website: https://cashapp.github.io/sqldelight/
+ *
+ */
+
+private val testUser by lazy { User(1L, "Jack") }
+private val testOrders by lazy {
+    listOf(
+            Orders(1L, "McDonalds", Date().time, testUser.user_id_pk),
+            Orders(2L, "Taco Bell", Date().time, testUser.user_id_pk),
+            Orders(3L, "Shake Shack", Date().time, testUser.user_id_pk)
+    )
```

Comment:
Too much indent here, please ensure you have configured the official Kotlin style guide in your IDE, and reformat the code with it (ctrl/cmd + alt/option + L by default).

Reply:
Done.
Sorry for that, My editor does not reformat those by default when wrapped by a lambda.